tests/app/main/test_services.py

```diff
@@ -287,9 +333,6 @@ def test_should_view_public_service_with_update_message(
             ]
 
         res = self.client.get('/suppliers/services/123')
-        if not framework_editable_services:
-            assert res.status_code == 404
```

Comment:
We will still get a 404 for DOS which has no `edit_service` manifest, but that's probably fine.